network-api/networkapi/wagtailpages/templates/wagtailpages/opportunity_page.html

```diff
@@ -35,6 +35,6 @@ <h1>{{ block.value }}</h1>
         {% endif %}
       {% endfor %}
     </div>
-    {% include "./fragments/ih_cta.html" %}
+    {% include "./fragments/ih_cta.html" with wrapper_class="my-5" %}
```

Comment:
out of scope for this PR (see other comments)

Reply:
see reasoning in https://github.com/mozilla/foundation.mozilla.org/pull/3066#discussion_r281411196